components/event-publisher-proxy/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-The Event Publisher Proxy receives Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway.
+The Event Publisher Proxy legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway.
```

Comment:
```suggestion
The Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway.
```

I don't understand why "receives" was removed, but there should be some kind of verb in the sentence explaining what the Event Publisher Proxy does.

Reply:
True, it was a typo. Fixed it.